src/store/middleware.ts

```diff
@@ -7,7 +7,7 @@ export const sagaMiddleware = createSagaMiddleware();
 const middleware = [sagaMiddleware];
 
 /* istanbul ignore next */
-if (process.env.NODE_ENV === 'development') {
+if (import.meta.env.DEV) {
```

Comment:
This env variable is provided by Vite by default: https://vitejs.dev/guide/env-and-mode.html#env-variables